### PR TITLE
1 deleted test

### PR DESCRIPTION
**Pull request by mumbler6**
Test was deleted in this commit `https://github.com/apache/ignite-3/commit/dd6ffb555b99d23b1484083e50831f934ee20405`

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -2383,7 +2383,7 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/configuration,org.apache.ignite.internal.configuration.util.ConfigurationUtilTest.testSchemaFields,ID,MovedToGradle,,https://github.com/apache/ignite-3/commit/c91f39cd80dfd143e5ac848486366ad6ed124ce5
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/page-memory,org.apache.ignite.internal.pagememory.freelist.AbstractFreeListTest.testMultiThread,ID,MovedToGradle,,https://github.com/apache/ignite-3/commit/c91f39cd80dfd143e5ac848486366ad6ed124ce5
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/runner,org.apache.ignite.internal.runner.app.IgniteCliRunnerTest.smokeTestArgs,ID,,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/runner,org.apache.ignite.internal.runner.app.IgniteCliRunnerTest.smokeTestArgsNullConfig,ID,,,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/runner,org.apache.ignite.internal.runner.app.IgniteCliRunnerTest.smokeTestArgsNullConfig,ID,Deleted,,https://github.com/apache/ignite-3/commit/dd6ffb555b99d23b1484083e50831f934ee20405
```
Comment (darko-marinov) [inline pr-data.csv:2386]: Skimming over https://github.com/apache/ignite-3/commit/dd6ffb555b99d23b1484083e50831f934ee20405 shows no changes in `IgniteCliRunner...`. What do you get with `git show --name-status dd6ffb555b99d23b1484083e50831f934ee20405 | grep IgniteCliRunner`.

Comment (mumbler6) [reply pr-data.csv:2386]: Sorry I forgot to mention this, they changed the name of the test file to IgniteRunnerTest.java in some commits before. I thought it was more appropriate to list the delete in this commit where you'll see the `smokeTestArgsNullConfig` is deleted.

Comment (darko-marinov) [reply pr-data.csv:2386]: Let's list both, first some `MovedOrRenamed` for the test class and then later `Deleted` for the test method.

Comment (mumbler6) [reply pr-data.csv:2386]: Do I make another entry in the csv with just the test class? With empty test method?

Comment (darko-marinov) [reply pr-data.csv:2386]: If the test class had the reported test method when the test class was moved, then keep the same class name and method name, just add `MovedOrRenamed` in the status (and the URL to the commit in `Notes`).

**Comment by darko-marinov** [comment]
You got `This branch has conflicts that must be resolved`. Also, it seem you claim the same test (same class and method name) was both `MovedOrRenamed` and `Deleted` -- both can't happen. Maybe the test was first moved/renamed to a different name, and then that different name was deleted.

**Comment by mumbler6** [comment]
Ok I understand what you mean. I've changed the deleted test to a different path, `IgniteRunnerTest.java`. I also just noticed another test I marked as movetogradle was also movedorrenamed, so I changed the status to that one instead.

**Comment by darko-marinov** [comment]
The branch cannot be rebased safely. Can you squash all your commits into one and rebase on the latest `main`.
